toolkit/components/uniffi-bindgen-gecko-js/src/render/cpp.rs

```diff
@@ -90,7 +86,9 @@ pub impl FFIFunction {
 #[ext(name=FFITypeCppExt)]
 pub impl FFIType {
     // Type for the WebIDL implementation method
-    fn type_name(&self) -> String {
+    //
+    // This is what we get passed from the JS code
```

Comment:
Is this better named something like `js_webidl_type`? Or maybe just a comment to indicate it's not actually a JS type? At least I hope that's what's going on :)  (That's really just a rhetorical suggestion)